sklearn/compose/_column_transformer.py

```diff
@@ -21,6 +21,7 @@
 from ..utils import Bunch
 from ..utils import _safe_indexing
 from ..utils import _get_column_indices
+from ..utils import _determine_key_type
```

Comment:
To fix the linting issue on the CI:

```suggestion
```

Reply:
Thanks @thomasjpfan for your feedback. I hope I have addressed them.